sysid-projects/drive/src/main/cpp/Robot.cpp

```diff
@@ -14,7 +14,6 @@
 #include <fmt/format.h>
 #include <frc/ADXRS450_Gyro.h>
 #include <frc/AnalogGyro.h>
-// #include <frc/romi/RomiGyro.h>
```

Comment:
Why is this commented out?

Reply:
We moved all the gyro setup code into `sysid-library` so it wasn't needed in the drivetrain project anymore.

Reply:
Should it be removed then?

Reply:
Isn't the git diff saying that it was removed?